README.md

```diff
@@ -73,5 +86,9 @@ Here comes the default configuration with some explanation:
 	-- will be closed. This makes most sense when paired with
 	-- `focus_on_open = false`
 	close_on_mouse_move = true,
+	-- When holding the cursor in the buffer, boo
```

Comment:
I don't really like the "in the buffer" part. Maybe removing it, or changing it to something like "over text" would make it a bit clearer.

Reply:
it should be `in_the_buffer` coz many buffers wont have `lsp's` like neo-tree nvim-tree oil etc